tests/test_service_catalog/test_forms/test_customer_request_forms_on_restricted_service.py

```diff
@@ -16,7 +16,7 @@ def setUp(self):
         self.service_test.billing_group_id = None
         self.service_test.save()
 
-    def test_create_request_without_billing_group_on_restricted_billing(self):
+    def test_cannot_create_request_without_billing_group_on_restricted_billing(self):
```

Comment:
Add a test: user try to request on restricted with a billing group where he not a member.